src/settings/OwnerSettings.js

```diff
@@ -18,7 +18,7 @@ class OwnerSettings extends React.Component {
     ownerServicePoints: {
       type: 'okapi',
       resource: 'service-points',
-      path: 'service-points',
+      path: 'service-points?limit=50',
```

Comment:
The `owners` resource accepts a limit of `500`. I gotta believe we want at least `100`, maybe `1000`.

Reply:
I have asked Holly, and current limit should be increased to 200. Thanks for review, Zak

Reply:
It is usually a good idea to ask about such things POs.